tensorflow_serving/model_servers/server_core.cc

```diff
@@ -291,6 +291,7 @@ FileSystemStoragePathSourceConfig ServerCore::CreateStoragePathSourceConfig(
     FileSystemStoragePathSourceConfig::ServableToMonitor* servable =
         source_config.add_servables();
     servable->set_servable_name(model.name());
+    servable->set_version_policy(model.version_policy());
```

Comment:
nit: set the fields in the same order they are defined in the proto (name, path, policy)